test/test_arhp.rb

```diff
@@ -52,48 +53,47 @@ def test_private_method
     assert Test1.connection.respond_to?(:test_private_method, true)
     refute Test1.connection.respond_to?(:test_private_method)
     assert_includes(Test1.connection.private_methods, :test_private_method)
-    assert(Test1.connection.send(:test_private_method) == true)
+    assert_equal Test1.connection.send(:test_private_method), true
```

Comment:
order is `expected`, `actual` ... so this is wrong ... prefer `Test1.connection.send(:test_private_method).must_equal true`